api/Document.json

```diff
@@ -8551,36 +8551,12 @@
                 "alternative_name": "mspointercancel"
               }
             ],
-            "firefox": [
-              {
-                "version_added": "59"
-              },
-              {
-                "version_added": "29",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.w3c_pointer_events.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "79"
-              },
-              {
-                "version_added": "29",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.w3c_pointer_events.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox": {
+              "version_added": "59"
+            },
+            "firefox_android": {
+              "version_added": "79"
```

Comment:
When was Firefox for Android 79 released? I think this is not old enough.

Reply:
Ah yeah, forgot to consider FxA's version -- do you think it makes more sense to keep only the desktop changes, or close this PR altogether and come back to it another time?

Reply:
First I would check if the Android data is actually correct, since the difference is so big. If it is I'd leave this cleanup for later, doing it all at once.